### ISSUE TITLE
Repository ercole is listed more than once in the configuration

### ISSUE THREAD

**Issue by sganderson** [ercole-io/ercole-docs#93]
After
`curl https://repository.ercole.io/shared/ercole-rhel8-x86_64.repo | tee /etc/yum.repos.d/ercole-x86_64.repo`
`curl https://repository.ercole.io/shared/ercole-rhel8-noarch.repo | tee /etc/yum.repos.d/ercole-noarch.repo`

OL8 yum install fails due to same name [ercole] for repository x86_64 and noarch

> yum install ercole ercole-web nginx
> Repository ercole is listed more than once in the configuration
> ercole-rhel8-noarch                                                                                                      17 kB/s | 4.4 kB     00:00
> No match for argument: ercole

**Comment by cirelli94** [comment]
Thanks for the issue @sganderson , I think you are right and only the repo for the right architecture must be added.
We are going to clarify this!

**Comment by amreo** [comment]
No, he needs either repositories because noarch contains only ercole-web packages, and x86_64 repo only ercole and ercole-agent...
I suspect something is misconfigured in ercole-web

**Comment by amreo** [comment]
I have found the solution of the issue.
Replace the first line of /etc/yum.repos.d/ercole-x86_64.repo to
```
[ercole-rhel8-x86_64]
```
And the first line of /etc/yum.repos.d/ercole-noarch.repo to
```
[ercole-rhel8-noarch]
```

@cirelli94 you need to fix the files in the repo (shared/*.repo)

**Comment by raffaelloferrari** [comment]
@sganderson 
We have some problems with the repository. If you want to be sure to install the latest versions available you can download them directly from https://repository.ercole.io/all/
The files you need for the server part are:
https://repository.ercole.io/all/ercole-2.4.4-1.el8.x86_64.rpm (backend)
https://repository.ercole.io/all/ercole-web-2.4.2-1.el8.noarch.rpm (frontend)
For the agent part:
https://repository.ercole.io/all/ercole-agent-rhel5-2.3.2-1.x86_64.rpm (rhel5)
https://repository.ercole.io/all/ercole-agent-2.3.5-1.el6.x86_64.rpm (rhel6)
https://repository.ercole.io/all/ercole-agent-2.3.5-1.el7.x86_64.rpm (rhel7)
https://repository.ercole.io/all/ercole-agent-2.3.5-1.el8.x86_64.rpm (rhel8)

**Comment by sganderson** [comment]
@raffaelloferrari Understood! I will update to latest version.
@amreo  the solution is correct i've done change manually first line

**Comment by raffaelloferrari** [comment]
@sganderson good!
I don't remember in which version we have changed the structure of the Mongo database so in case the ercole service doesn't start correctly you have to run the command `ercole migrate`
And if you want to see the data of the licenses as per the Oracle list with part number and description let us know and we will share you a collection to import in the database with the ready data. 
![image](https://user-images.githubusercontent.com/47225245/125844131-480c1c2b-dfda-4f25-a644-1e2b62bfbd21.png)

**Comment by sganderson** [comment]
@raffaelloferrari  i confirm service doesn't start and require `ercole migrate` command.
I'm trying windows agent but i'm stuck here 
[21-07-16 09:40:20][AGEN][INFO][/root/project/main.go:94] Sending data...
[21-07-16 09:40:20][AGEN][WARN][/root/project/main.go:132] Response status: 405 Method Not Allowed
[21-07-16 09:40:20][AGEN][WARN][/root/project/main.go:133] Sending result: FAILED

And on server side
Jul 16 09:40:20 ercole02 ercole[17312]: [21-07-16 09:40:20][DATA][INFO][/root/project/utils/loggingHandler.go:31] HTTP Request                                      endpoint=GET /hosts statusCode=405 accept= clientSocket=127.0.0.1:41580 serverSocket=127.0.0.1:11111 size=0 timeSta>
Jul 16 09:40:41 ercole02 ercole[17312]: [21-07-16 09:40:41][DATA][INFO][/root/project/utils/loggingHandler.go:31] HTTP Request                                      endpoint=GET / statusCode=404 accept=text/html,application/xhtml+xml,application/xml;q=0.9,image/avif,image/webp,im>
Jul 16 09:41:08 ercole02 ercole[17313]: [21-07-16 09:41:08][APIS][INFO][/root/project/utils/loggingHandler.go:31] HTTP Request                                      endpoint=GET /hosts?mode=summary statusCode=200 accept=application/json, text/plain, */* clientSocket=127.0.0.1:549>

someting is missing?

**Comment by cirelli94** [comment]
If you want you can let `ercole` do the migration automatically, setting in your `config.toml`:
```
[Mongodb]
...
Migrate = true
```
But if you want to known a migration is coming and to do a backup first, you can leave it at false :)


As regards your error, I'm pretty sure you are trying to send data to a wrong service of Ercole: you have to the `DataserviceURL` parameter on `config.json` on your windows agent installation to the right URL.
If you have followed our installation instructions, it must be something like:
```
    "DataserviceURL": "https://myercole.local/data",
```

**Comment by sganderson** [comment]
Nice! 
I set `DataServiceURL` correctly but i receive error 422 and i reproduce using `ercole fire-hostdata`

File: json.data Status: 422 Cause: {"error":"Invalid hostdata: - features.oracle: Must validate at least one schema (anyOf). Value: [map[database:map[databases:[map[addms:[] allocable:129 archivelog:false asm:fa..]\n- features.oracle.database: Must validate at least one schema (anyOf). Value: [map[databases:[map[addms:[] allocable:129 archivelog:false asm:false backups:[..]\n- features.oracle.database.databases.0.patches.0.date: Does not match format 'date'. Value: []\n","message":"Unprocessable Entity","sourceFilename":"/root/project/data-service/controller/hosts.go","lineNumber":43}

**Comment by cirelli94** [comment]
We are going OOT here, would you mind to talk about it on https://gitter.im/ercole-io/community ?
Thanks
